packages/block-library/src/query-pagination/block.json

```diff
@@ -4,6 +4,7 @@
 	"category": "design",
 	"usesContext": [ "queryId", "query" ],
 	"supports": {
+		"align": true,
```

Comment:
This is not needed right?

Reply:
This not related to the Query change, but isn't it needed for adding align support in QueryPagination?

Reply:
If we do want to support alignment there yes. Was this a requirement?

Reply:
I believe it's needed as is closely related to Query 🤔. No strong opinions though.

--cc @kjellr , @carolinan, @aristath

Reply:
I think we should support alignment there. But I don't know that it needs to happen in this PR specifically.

Reply:
What Kjellr said, there needs to be alignment options for the pagination, but it can be handled separately.